autoloader.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * We handle here two possible locations of composer-generated autoload file. One is directly in Elgg directory
+ * when not using as dependency, other is in the main project dir, parent to this one.
+ */
+
+$paths = [
+	__DIR__ . '/vendor/autoload.php',
+	__DIR__ . '/../../../vendor/autoload.php',
```

Comment:
I wanted to avoid having this additional autoloader.php, but that sees to be the way other packages handle 2 scenarios of autoloading as well.